src/calculateRentalCost.js

```diff
@@ -1,10 +1,25 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
The function name 'calculateRentalCost' is descriptive and clearly indicates its purpose, which is good. However, the parameter 'days' could be more descriptive, such as 'numberOfDays', to better represent its role in the calculation.